src/components/toolbar/index.js

```diff
@@ -26,6 +26,47 @@ class Toolbar extends Component {
     super(props)
   }
 
+  isFocused = () => {
+    var elem = document.activeElement
+    do {
+      if (elem.matches('div.villain')) {
+        return true
+      }
+    } while ((elem = elem.parentElement) !== null)
+    return false
+  }
+
+  handleShortcuts = event => {
+    if (!this.isFocused()) {
+      return
+    }
+
+    const { toggleSetting, navigateToPage } = this.context
+    const { isFirstPage, isLastPage, currentPage } = this.context.state
+
+    switch (event.keyCode) {
+      case 70: //Key: f
```

Comment:
why not using the `key` property from the event instead of the `keyCode`, would be more readable IMO.
https://fr.reactjs.org/docs/events.html#keyboard-events

Reply:
great idea :+1: